README.md

```diff
@@ -40,64 +40,141 @@
 </div>
 
 ## Table of Contents
-- [API](#api)
-- [Installation](#installation)
 - [Development](#development)
-- [Environment](#environment)
+- [API](#api)
 - [Testing](#testing)
 - [Commands](#commands)
 - [Code style](#code-style)
 - [Contributors](#contributors)
 - [See Also](#see-also)
 
-## API
+## Getting Started
 
-If you want to build on the API for personal use, take a look through the work done in the [Migrate to new APIs project](https://github.com/resonatecoop/stream/projects/5#card-59829409) and consider checking the [backlog in our community forum](https://community.resonate.is/c/platform/52). 
+### Quickstart - no HTTPS
+
+Quick-n-dirty instructions to get the player up and running on your computer using http and pointing to the existing production API (see [API](#api) to learn more about the API). 
+Assumes the latest version of [node.js](https://nodejs.org/).
 
-## Installation
+_Stuck or curious about the roadmap? Ask for help in the [forum](https://community.resonate.is/t/development-team/1724)_. 
 
-First, make sure you have the latest version of [node.js](https://nodejs.org/)
+Clone the repo and `cd` into it:
 
-To use this project you also need [lerna](https://github.com/lerna/lerna). Lerna is a tool for managing JavaScript projects with multiple packages. .
+```sh
+git@github.com:resonatecoop/stream.git
+cd stream
+```
+
+Install [lerna](https://github.com/lerna/lerna). Lerna is a tool that optimizes the workflow around managing multi-package repositories with git and npm. 
 
 ```sh
 npm i lerna -g
 ```
 
-Then you should install dependencies using the bootstrap command.
+Install dependencies: 
 
 ```sh
 lerna bootstrap
 ```
 
-## Development
+Create your env file:
 
-### Beta app
+```sh
+cp beta/.env.example beta/.env
+```
+
+You may point your `.env` file to the production API:
+
+```
+API_DOMAIN=api.resonate.coop
+APP_DOMAIN=beta.stream.resonate.coop
+APP_HOST=https://beta.stream.resonate.coop
+APP_PORT=""
+```
 
-To start beta app with `bankai`.
+Run the app:
 
 ```sh
-npm start
+npm run dev
 ```
 
-By default, the app is accessible at `https://localhost:8080`.
+You should see the app running at http://localhost:8080.
 
 You can run the app at a different port using the command below:
 
 ```sh
 npm run dev -- -- --port 8089
 ```
 
-See [bankai](https://github.com/choojs/bankai) docs for usage.
+The embed app (optional)
 
-### Other apps (embed, upload, ...)
+```sh
+lerna run --scope embed start --parallel
+```
+
+### Using HTTPS
+
+You will notice the app does not work well without HTTPS. Secure connection is required to fully test PWAs.
+
```

Comment:
@auggod What is the rationale for effectively requiring https in local development? This will turn alot of devs off. We can offer this as an option, but it shouldn't be required. Are there any barriers to supporting http development?

Note that https is not required to use service workers locally. See further https://stackoverflow.com/questions/34160509/options-for-testing-service-workers-via-http

Reply:
Regarding `Secure connection is required to fully test PWAs.` even if the hard-coded https were removed, my understanding is that for the user to test on real mobile devices we'd need the https, so this part about "fully test" is still true. 

Perhaps it would be better to have a "testing on mobile" section explaining how to do that?

Reply:
Yeah, currrently it reads like you have to have https setup to develop. Let's make it clear https is only necessary to test the PWA on mobile.

Reply:
the developer is going to quickly run into CORS issues if they don't follow all the steps in the readme - I'd recommend doing the HTTP steps just to sanity check that most things are working, but once they start trying to do things like view and edit playlist they're gonna get stuck.

Reply:
@whatSocks hm  there isn't any real reason to require https in this scenario as all we're doing is getting data. Asking folks to setup https for development is a significant barrier. 

@auggod how about we just add localhost as ``Access-Control-Allow-Origin`` for CORS on GET requests? We're not getting any additional security benefits here by enforcing it in this scenario.

Reply:
I'm able to use all features on http - I think we can assume an http-only user should be able to run the frontend.